server/controllers/stock/reports/purchaseOrderAnalysis/report.handlebars

```diff
@@ -72,69 +73,70 @@
 
         {{#if this.displayDetails }}
         {{#each this.inventoriesOrdered}}
-          <br>
-          <br>
-          <table style="width: 90%; margin-right: auto; margin-left: auto" class="table table-striped table-condensed table-report table-bordered">
-
-              <tr>
-                <td width="15%" class="text-uppercase"> {{translate 'FORM.LABELS.CODE'}}  </td>
-                <td colspan="7" width="85%"><strong> {{ code }} </strong></td>
-              </tr>
-              <tr>
-                <td width="15%" class="text-uppercase"> {{translate 'FORM.LABELS.INVENTORY'}}  </td>
-                <td colspan="7" width="85%"><strong> {{ inventory_text }} </strong></td>
-              </tr>
+          <table class="table-condensed table-report table-bordered">
```

Comment:
We need to have some sort of avoiding page break in here.   Otherwise, we lose information:

![image](https://user-images.githubusercontent.com/896472/87655881-aeab6280-c750-11ea-9999-a3484a19f99e.png)

Reply:
> We need to have some sort of avoiding page break in here. Otherwise, we lose information:
> 
> ![image](https://user-images.githubusercontent.com/896472/87655881-aeab6280-c750-11ea-9999-a3484a19f99e.png)

How do you think we can fix this behavior in this PR cause I don't know since we have a variable table number and variable rows for each table